src/Microsoft.Identity.Abstractions/TokenAcquisition/AcquireTokenOptions.cs

```diff
@@ -90,6 +91,11 @@ public AcquireTokenOptions(AcquireTokenOptions other)
         /// </summary>
         public string? PopClaim { get; set; }
 
+        /// <summary>
+        /// Specifies if the client credential is a managed identity.
+        /// </summary>
+        public string? ManagedIdentity { get; set; }
```

Comment:
This should not be a `string?` but a `ManagedIdentityOptions?`

Reply:
fixed